### PR TITLE
KEYCLOAK-17693 add config for loading custom IdMapper class

### PR DESCRIPTION
**Pull request by Yang-Xie-OSS**
The documentation of PR [KEYCLOAK-17693](https://github.com/keycloak/keycloak/pull/7921).
And this is the [JIRA](https://issues.redhat.com/browse/KEYCLOAK-17693) link.

### CODE REVIEW

**Review by mposolda — DISMISSED** [review]
@Yang-Xie-OSS Thanks! I am approving. IMO this can be merged once the codebase is merged (which depends on the tests).

@andymunro Do you please have a chance to review this?

File: securing_apps/topics/oidc/java/servlet-filter-adapter.adoc [hunk 1]
```diff
@@ -52,6 +52,16 @@ Note that you should configure your client in the {project_name} Admin Console w
 The Admin URL will make callbacks to the Admin URL to do things like backchannel logout.
 So, the Admin URL in this example should be `http[s]://hostname/{context-root}/keycloak`.
 
+If you need to customize the session ID mapper you can configure the fully qualified name of the class in the Filter init-param keycloak.config.idMapper. Session ID mapper is a mapper which is used to map user IDs and session IDs. By default org.keycloak.adapters.spi.InMemorySessionIdMapper is configured.
```
Comment (andymunro) [inline securing_apps/topics/oidc/java/servlet-filter-adapter.adoc]: comma after an introductory clause -- after mapper,
"is a mapper that..." - use "that' not "which" when the rest of the sentence qualifies the noun, meaning it is the specific mapper "that" is used to map user IDs and session IDs.

**Review by andymunro — CHANGES_REQUESTED** [review]
Some minor changes.

**Comment by Yang-Xie-OSS** [comment]
@andymunro 
Thanks for the review. I've edited it.

**Review by mposolda — APPROVED** [review]

**Pull request merged**